src/deno/stable/functions.ts

```diff
@@ -58,6 +58,7 @@ export { removeSync } from "./functions/removeSync.js";
 export { rename } from "./functions/rename.js";
 export { renameSync } from "./functions/renameSync.js";
 export { shutdown } from "./functions/shutdown.js";
+export { sleepSync } from "./functions/sleepSync.js";
```

Comment:
It was intentionally unexported

https://github.com/denoland/deno.ns/blob/88aafe7668df42075e7cd3ed9736e03de4f3d877/src/deno/stable/functions/sleepSync.ts#L3

Please refer to PR: https://github.com/denoland/deno.ns/pull/18